tests/integration-tests/tests/auth.setup.ts

```diff
@@ -0,0 +1,31 @@
+import { test as setup } from '@playwright/test';
+import path from 'path';
+
+const authFile = path.join(__dirname, '../playwright/.auth/user.json');
+
+setup('authenticate', async ({ page }) => {
+
+  const BASE_URL = process.env.BASE_URL;
+  const EMAIL = process.env.EMAIL;
+  // decode the password
+  const PWD = Buffer.from(process.env.PASSWORD_BASE64 || '', 'base64').toString('utf-8');
+  if (!BASE_URL || !EMAIL || !PWD) {
+    throw new Error('BASE_URL, EMAIL or PWD is not set');
+  }
+  await page.goto(BASE_URL);
+  await page.getByRole('textbox', { name: 'Email' }).click();
```

Comment:
you could store the textbox as a reference e.g. emailTextbox since you're referencing it multiple times.